test/opentok-test.js

```diff
@@ -51,6 +51,22 @@ var mockBroadcastObject = {
   status: 'stopped'
 };
 
+var mockListBroadcastsObject = {
+  "count": 1,
+  "items": [{
+    id: 'fooId',
+    sessionId: 'fooSessionId',
+    projectId: 1234,
+    createdAt: 1537477584724,
+    updatedAt: 1537477584725,
+    broadcastUrls: { hls: 'hlsUrl' },
+    maxDuration: 7200,
+    resolution: '1280x720',
+    event: 'broadcast',
+    status: 'stopped'
+  }]
+};
+
 function validateBroadcastObject(broadcast, status) {
   expect(broadcast.id).to.equal('fooId');
   expect(broadcast.id).to.equal('fooId');
```

Comment:
You can remove this line.

Reply:
Hmm, I'm just leaving a line between the next function. I think it improves readability and it's how all of the other functions are written.

Reply:
The following line is included twice: `expect(broadcast.id).to.equal('fooId');`. We can remove one of them.

Reply:
Ah great catch - fixed!